forte/data/extractor/base_extractor.py

```diff
@@ -231,16 +231,25 @@ def extract(self, pack: DataPack,
         """
         pass
 
-    def remove_from_pack(self, pack: DataPack, instance: Annotation):
-        r"""Functionality: Remove the existing feature of the instance
+    def pre_evaluation_action(self, pack: DataPack,
+                instance: Annotation):
+        r"""Remove the existing feature of the instance
         in a pack. We might remove an attribute under an entry or remove
         the entry itself directly, which will depend on the different
-        type of extractors.
+        type of extractors. This function is called before the evaluation
+        on a pack to do some pre_evaluation action. For example, you can
+        remove entries or remove some attributes of the entry. So that
+        they are not mixed with predicted entries or predicted attributes.
 
         Overwrite instruction:
```

Comment:
I don't think you need this instruction. You just need to tell users what this function should do.

Reply:
Got it.